tools/cli/main.js

```diff
@@ -746,6 +746,44 @@ Fiber(function () {
     rawArgs.push(term);
   }
 
+  // Prevent running meteor as root
+
+  var unixPlatforms = {
+    'darwin': true,
+    'linux': true,
+    'freebsd': true
+  };
+  var platform = process.platform;
+
+  if (_.has(unixPlatforms, platform)){
+    // On UNIX platforms.
+    if (process.getgid() === 0 && process.getuid() === 0){
```

Comment:
Please just check `process.getuid() === 0`.  There is no need to check `process.getgid()` as well since uid/gid is always 0/0 for `root`.